sql-schema/282.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO `widgets` (`widget_title`, `widget`, `base_dimensions`) VALUES ('Alert History','alertlog',"6,3");
```

Comment:
Hi @louis-oui 
You cannot use the xxx.sql files anymore. Changes are now using Laravel migrations.

Reply:
Done in database/seeds/DefaultWidgetSeeder.php but it does not appear into database. I don't know how to make new entry to appear

`select * from widgets;`

Reply:
You need to create a migration file in ```./database/migrations```. These file basically describe what to do to migrate, and what to do to revert.

Reply:
Actually, it is not a database table migration.

It is an insert in a table.

Referring to https://github.com/librenms/librenms/pull/10541, I did the correct thing but I need to do :
`php artisan db:seed`

But this command is not present in "Advanced users" section of https://docs.librenms.org/General/Updating/

Is seed done by daily.sh ?

Reply:
DB seed is only done on a fresh install. As far as I see it, any other migration is only taken into account during updates (daily) via migrations. @murrant ?

Reply:
I don't see any place where db seeds is done on daily.sh
I suggest fixing it : https://github.com/librenms/librenms/pull/10917

Reply:
I am not familiar with Laravel framework enough to merge #10917. We can still merge this PR (but the new widget won't be available until seed is manually run or 10917 is merged.)
@librenms/reviewers , if anybody is comfortable with this topic, please jump in.

Reply:
I fixed https://github.com/librenms/librenms/pull/10917 to only update widget seeds

Reply:
I think that the feature is ready to be merged now with https://github.com/librenms/librenms/pull/10917